snapcraft.yaml

```diff
@@ -1393,6 +1393,15 @@ parts:
       go build -o "${CRAFT_PART_INSTALL}/bin/lxd-benchmark" github.com/canonical/lxd/lxd-benchmark
       go build -o "${CRAFT_PART_INSTALL}/bin/lxd-user" github.com/canonical/lxd/lxd-user
 
+      # Build the static website
+      apt install -y shellcheck python3.10-venv
+      make doc
+      rm -Rf doc/html/.doctrees
+
+      # Copy the static website
+      mkdir -p "${CRAFT_PART_INSTALL}/share/lxd-documentation"
+      cp -a doc/html/. "$CRAFT_PART_INSTALL/share/lxd-documentation/"
```

Comment:
does the lxd-ui part put the `html` dir inside `share/lxd-ui` or does it put the html files directly into `share/lxd-ui`?

Reply:
directly inside `share/lxd-ui`

Reply:
Hence why we do the same here